app.js

```diff
@@ -1,12 +1,14 @@
 const express = require("express");
-const { getTopics } = require("./controller");
+const { getTopics, getEndpoints } = require("./controller");
 
 const app = express();
 
 app.use(express.json());
```

Comment:
Make sure not to include code you aren't using (in this case `app.use(express.json())`